Makefile

```diff
@@ -8,7 +8,7 @@ else
 	GOOS=linux
 endif
 
-DEV_ENV_IMAGE := quay.io/deis/go-dev:0.11.1
+DEV_ENV_IMAGE := quay.io/deis/go-dev:v1.27.1
```

Comment:
Can you change to use our own image here: 

```
hephy/go-dev:v1.26.2
```

We are building it from here and merging upstream changes: https://github.com/teamhephy/docker-go-dev

Reply:
Pushed the change.

Reply:
Thank you for making this small change @jfuechsl!